ssl/test/runner/runner.go

```diff
@@ -3045,14 +3045,13 @@ read alert 1 0
 			expectedLocalError: "remote error: illegal parameter",
 		},
 		{
-			name: "TooManyKeyUpdates",
+			name: "LotsOfKeyUpdates",
```

Comment:
Should we leave a comment above here, describing that this isn't expected behavior when using AWS-LC, but we still allow it anyways and are testing it?